wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ViewModel/ConversationListViewControllerViewModel.swift

```diff
@@ -233,6 +240,16 @@ extension ConversationListViewController.ViewModel: ZMInitialSyncCompletionObser
     }
 }
 
+// MARK: - Observing UIApplication Life Cycle
+
+extension ConversationListViewController.ViewModel {
+
+    @objc
+    private func handleApplicationDidBecomeActiveNotification(_ notification: Notification) {
```

Comment:
Why do you want to have `notification` parameter ?

Reply:
"The method that aSelector specifies must have one and only one argument (an instance of NSNotification)."
https://developer.apple.com/documentation/foundation/notificationcenter/1415360-addobserver